workspaces/marketplace/plugins/marketplace-common/src/types.ts

```diff
@@ -84,6 +92,36 @@ export interface MarketplacePluginSpec extends JsonObject {
     appconfig?: string;
   };
 }
+
+/**
+ * @public
+ */
+export interface MarketplacePackageSpec extends JsonObject {
+  packageName: string;
+  dynamicArtifact?: string;
+  author?: string;
+  support?: string;
+  lifecycle?: string;
+  backstage?: MarketplacePackageBackstage;
+  appConfigExample?: AppConfigExample;
```

Comment:
Could we have an array here? For the case that a plugin/package author wants provide multiple examples.

Like "Configure a Kubernetes cluster"  and "Configure an OpenShift cluster"

```suggestion
  appConfigExamples?: AppConfigExample[];
```